conf/modules.config

```diff
@@ -238,6 +238,15 @@ process {
         ext.prefix = { "${report}.rgi" }
     }
 
+    withName: HAMRONIZATION_FARGENE {
+        publishDir = [
+            path: { "${params.outdir}/arg/hamronization/fargene" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+        ext.prefix = { "${meta.id}_${report}.fargene" }
```

Comment:
I think I need example output too see why we need `${meta.id}_` here for fARGene but not for the other hAMRonization tools? :eyes: